modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -1026,6 +1028,56 @@ public void testPutObjectDefinition() throws Exception {
 		_objectDefinitionLocalService.deleteObjectDefinition(
 			randomModifiableSystemObjectDefinition.getId());
 
+		// Modifiable system object definition different language ID
+
+		randomModifiableSystemObjectDefinition =
+			objectDefinitionResource.postObjectDefinition(
+				_randomModifiableSystemObjectDefinition());
+
+		objectDefinitionDefaultLanguageId = "pt_BR";
+
+		siteDefaultLanguageId1 = LanguageUtil.getLanguageId(
+			LocaleUtil.getSiteDefault());
+
+		Assert.assertNotEquals(
```

Comment:
Does this assert makes sense for your scenario or did you just copy and paste the code above?